### PR TITLE
created backend skeleton

### PR DESCRIPTION
**Pull request by sophieeejw**
Relates #5

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+app.use((err,req,res,next)=>{
+  console.log(err.stack);
+  res.status(500).send("Internal Server Error");
+});
+app.use(router);
+app.set('port', process.env.PORT || 3000);
+
+app.listen(app.get('port'),()=>{
+  console.log(`We are listeining now at http://${process.env.HOST ? process.env.HOST : "localhost"}:${app.get('port')}`);});
```
Comment (claireinez) [inline src/server.js]: "listening" is spelled wrong.

File: src/router.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  res.send('<h1>You are in the homepage</h1>');
+})
+
+router.get("*", (req, res)=> {
+  console.log(req.url);
```
Comment (claireinez) [inline src/router.js]: Is this `console.log` needed in source control?

Comment (sophieeejw) [reply src/router.js]: log deleted!

File: src/server.js (hunk 1 above)
Comment (sophieeejw) [reply src/server.js]: spelling corrected

**Comment by sophieeejw** [comment]
@claireinez reflected your feedback, please review it again :)

**Comment by claireinez** [comment]
In future, please use "Closes #5" as that is a GitHub keyword and it will automatically close the issue for us 😃

**Pull request merged**